src/libexpr/primops.cc

```diff
@@ -3770,6 +3857,15 @@ static RegisterPrimOp primop_elem({
     .doc = R"(
       Return `true` if a value equal to *x* occurs in the list *xs*, and
       `false` otherwise.
+
+      # Time Complexity
+
+      O(n * T) (worst case) where:
+
+      n = list length
+      T = time to compare average element
```

Comment:
`O()` expresses an upper bound, so this should be the upper bound too.
Let's not get into average time.
We can use vagueness to our advantage though:
```suggestion
      T = time to compare two elements
```

This is by `==`, right?